src/extension/commandPaletteHandler.ts

```diff
@@ -110,7 +110,7 @@ export class CommandPaletteHandler {
                     // Execute the operation
                     return operation();
                 } else {
-                    vscode.window.showErrorMessage("Current workspace is not a React Native project.");
+                    // vscode.window.showErrorMessage("Current workspace is not a React Native project.");
```

Comment:
We recently made a change so that `isReactNativeProject` is much more specific, and should only report false if we cannot run react-native commands. After you merge with latest master, could you check if this still reports errors too often for you?